main/sdr.go

```diff
@@ -150,9 +139,17 @@ func (e *ES) sdrConfig() (err error) {
 	return
```

Comment:
Suggest to add `log.Printf("===== ES Device Serial: %s PPM %d =====\n", e.serial, e.ppm)`